### PR TITLE
fixed turn system

### PR DESCRIPTION
**Pull request by gaguevaras**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -2,16 +2,17 @@ var IRCConnector = require("./classes/IRCConnector.class");
 var JSONConnector = require("./classes/persistence/JSONConnector.class");
 
 // Sample Slack-thru-IRC config
-/*module.exports = {
+module.exports = {
```
Comment (slashman) [inline config.js]: Let's not commit config.js unless we add new params or something

Comment (gaguevaras) [inline config.js]: Commiting due to your modification to use the IRC config, I test on Slack.

Comment (slashman) [inline config.js]: May be those modifications should be kept local? I test mostly outside slack so the file will keep bouncing back and forth... let's discuss it... I'll merge for now.

Comment (gaguevaras) [inline config.js]: Agreed, the configuration is dependent on the runtime environment anyway, good call!

**Pull request merged**